substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -95,6 +95,8 @@
     private static final String DEFAULT_GENERATOR_9PLUS_SUFFIX = "$JDK9Plus";
     private static final String CUSTOM_SYSTEM_CLASS_LOADER = NativeImageSystemClassLoader.class.getCanonicalName();
 
+    private static final int WINDOWS_MAX_COMMAND_LENGTH = 8192;
```

Comment:
You could make this something closer to the [real limit](http://msdn.microsoft.com/en-us/library/ms682425%28VS.85%29.aspx) of 32768.

Reply:
I wasn't actually sure what the real limit was. Google turned up 2047 for some Windows versions and 8191 for others (see https://support.microsoft.com/en-us/help/830473/command-prompt-cmd-exe-command-line-string-limitation). THe number of 32768 is also mentioned but for the size of the ENV vars.
If you are sure it's 32768 I'll be more than happy to change it.

Reply:
Ah duh, I see you actually provided a link. Sorry about the noise, I'll change it ASAP :-)

Reply:
Done.